File: src/go/classes.go
package main

import "fmt"

type MontyPython struct {
	FirstName 	string
	LastName 	string
}

func (montyPython *MontyPython) FullName() string {
	return montyPython.FirstName + " " + montyPython.LastName
}

func main() {
	montyPython := MontyPython{"John", "Cleese"}
	fmt.Println("Monty Python is " + montyPython.FullName() + ".")
	
	montyPython.FirstName = "Eric"
	montyPython.LastName = "Idle"
	fmt.Println("Now, Monty Python is " + montyPython.FullName() + ".")
}

File: src/rb/classes.rb
class MontyPython
    def initialize(firstName, lastName)
        @firstName = firstName
        @lastName = lastName
    end

    def FirstName=firstName
        @firstName = firstName
    end

    def LastName=lastName
        @lastName = lastName
    end

    def FullName
        return @firstName + " " + @lastName
    end
end

montyPython = MontyPython.new("John", "Cleese")
puts "Monty Python is #{montyPython.FullName}."

montyPython.FirstName = "Eric"
montyPython.LastName = "Idle"
puts "Now, Monty Python is #{montyPython.FullName}."